database.py

```diff
@@ -19,25 +36,13 @@ def select(c, query, returnJsonFormat=True):
         return json.dumps(records)
     return records
 
-def select_by_id(c, query, id, returnJsonFormat=True):
-    c.execute(query, (id, ));
-    record = c.fetchone()
-    if (returnJsonFormat):
-        return json.dumps(record)
-    return record
+def delete_by_id(c, tableName, id):
+    c.execute('DELETE FROM ' + tableName + ' WHERE ROWID =(?)', id)
 
-def run_migrations():
-    # print('Database created')
-    cursor, conn = connect() 
 
-    #create table
-    cursor.execute('''CREATE TABLE IF NOT EXISTS ContactList
-                ( name TEXT NOT NULL,  
-                phoneNumber TEXT NOT NULL UNIQUE,
-                email TEXT NOT NULL UNIQUE)''')
+def update_contacts(c, tableName, id, phone):
```

Comment:
Since this function only updates the phone number, it'd be better if the name would represent that.

Additionally, since we are only updating the phone number, and the phone number is always stored in `ContactList`, we should not need to give it the table name. 

In general, the fewer arguments (also called parameters) we have, the better. 

Tip: You already applied this feedback in the `delete_by_id` function.

```suggestion
def update_phone_number(c, id, phone):
    c.execute('UPDATE ContactList SET phoneNumber = ? WHERE ROWID = ?', (phone, id))
```